### DISCUSSION TITLE
Fraction heights not working for me

### DISCUSSION THREAD

**Discussion by Sajid2001** [tailwindlabs/tailwindcss#7680] [Help]
Here is my code in react js. I am trying to crop out a certain fraction of the image but all the h-fraction class names (h-1/2, h-2/3, etc) are not working for me. 

```
<div className=" mt-4 mx-3 flex h-full bg-white rounded overflow-hidden shadow-lg">
        <div className="flex flex-wrap no-underline">
            <div className="w-full h-3/5 md:w-full rounded-t">
                <img src = {url}></img>
            </div>
        </div>
    </div>
```

**Comment by adamwathan** [comment]
Hey! The classes definitely generate the correct CSS, the problem is more just in how you are trying to use them.

Setting `height: full` in CSS doesn't really do anything unless the parent element has a defined height, which is maybe something that has been left out of your example.

Here you can see that as long as there's a parent element with a height, the fraction height is applied to the image wrapper as expected:

https://play.tailwindcss.com/VfQhKtSlkv

Relative heights in general are kinda tricky in CSS, hope that helps.

**Reply by Sajid2001** [reply]
So I need a div that sets a height, then have a child div use h-full, and then have the child of that div use h-2/3?
